packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -980,15 +981,22 @@ export class LitContracts {
     return data;
   }
 
+  /**
+   * Mints a new token with authentication.
+   *
+   * @param authMethod - The authentication method.
+   * @param scopes - The permission scopes.
+   * @param pubkey - The public key.
+   * @param authId - (optional) The authentication ID.
```

Comment:
Why is authId separate from authMethod? Isn't it included in the authMethod object?

Reply:
We might have different terms being used in the SDK and the node?
<img width="384" alt="image" src="https://github.com/LIT-Protocol/js-sdk/assets/4049673/0c87dc7b-d819-48a0-9acc-b4ac08045324">